catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/OverrideAttributesSupport.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package ddf.catalog.impl.operations;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import ddf.catalog.content.data.ContentItem;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.impl.MetacardImpl;
+
+public class OverrideAttributesSupport {
+    private OverrideAttributesSupport() {
+
+    }
+
+    public static void overrideAttributes(List<ContentItem> contentItems,
+            Map<String, Metacard> metacardMap) {
+        for (ContentItem contentItem : contentItems) {
+            if (contentItem.getMetacard() == null) {
+                continue;
+            }
+
+            Metacard currentMetacard = metacardMap.get(contentItem.getId());
```

Comment:
If you extract the code from here to line 50 into a public method that takes two metacards, I think you might be able to remove the duplicate code from `CreateRequest.populateMetacardMap` and `UpdateResponseupdate`.